app/controllers/idv/enter_password_controller.rb

```diff
@@ -1,14 +1,13 @@
 module Idv
   class EnterPasswordController < ApplicationController
-    before_action :personal_key_confirmed
-
     include IdvStepConcern
     include StepIndicatorConcern
     include PhoneConfirmation
     include FraudReviewConcern
 
     before_action :confirm_verify_info_step_complete
     before_action :confirm_address_step_complete
+    before_action :confirm_no_profile_yet
```

Comment:
How/why would a user not have a profile at this stage? (How would I go about testing this?) I just can't think of it at the moment.

Reply:
@soniaconnolly Hmm, `:confirm_idv_needed` is already in the chain, since this controller includes `IdvStepConcern`. `confirm_no_profile_yet` skips past the password controller if the user already has a profile (except in an edge case around IPP). Maybe `skip_step_if_possible` is a better name for that `before_action`.

@gina-yamada Most users would not have a profile here since `init_profile` (called during `create`) creates them.